### DISCUSSION TITLE
Availability of Larval Connectivity Tools

### DISCUSSION THREAD

**Discussion by ZoeZhouJ** [jjrob/MGET#16] [General]
Hi,
Really exited for this development. I'm checking if the Larval Connectivity modeling tools are available in the current MGET version for Python 3.x and ArcGIS Pro. Thanks!

**Comment by jjrob** [comment]
Hi, thanks for your interest. The larval connectivity modeling tool is not yet available. We have done some of the big prerequisites to getting it working, including the steps needed to interface MGET with the MATLAB Runtime via the MATLAB Compiler, and developing a tool to access the new Python API of Copernicus Marine Service, which is a good provider of ocean currents data. However, we have not ported the connectivity tool itself yet.

We have a big cetacean species distribution modeling project coming up and need to focus on that for a while, so we were not planning to get to the connectivity tool until sometime in 2025. However, the connectivity tool is a popular part of MGET and you are not the only person to inquire about it recently. Can you tell me a bit about the project you're contemplating and its rough timeframe, if you have one in mind? Perhaps there is a way that we could get to this sooner rather than later, particularly if you were ready to test it out on a real project. Currently our work is focused on marine mammal SDMs, so we don't have a "real" connectivity project that is active right now.
